### PR TITLE
Use a minimal Redux logger during development on the server

### PR DESCRIPTION
**Pull request by kumar303**
Fixes https://github.com/mozilla/addons-frontend/issues/6722

I think this will be useful. Example:

screenshot 2018-10-23 12 48 53

### CODE REVIEW

File: src/core/store.js [hunk 1]
```diff
@@ -3,26 +3,40 @@ import config from 'config';
 import { applyMiddleware, compose } from 'redux';
 import { createLogger } from 'redux-logger';
 
+import log from 'core/logger';
+
+export const minimalReduxLogger = () => (next) => (action) => {
```
Comment (bobsilverberg) [inline src/core/store.js]: The changes seem fine to me, but I don't really understand everything that is going on. What's actually happening here with this `next` and `action`?

File: tests/unit/core/test_store.js [hunk 2]
```diff
@@ -87,4 +107,20 @@ describe(__filename, () => {
     expect(typeof middleware({ _config, _window })).toBe('function');
     sinon.assert.notCalled(_window.__REDUX_DEVTOOLS_EXTENSION__);
   });
+
+  describe('minimalReduxLogger', () => {
+    it('process an action', () => {
```
Comment (bobsilverberg) [inline tests/unit/core/test_store.js]: I don't really understand what's going on in this test either. I've read through it a few times, but there's always a point at which I get lost. :-(

File: src/core/store.js (hunk 1 above)
Comment (kumar303) [reply src/core/store.js]: The signature of Redux middleware is to call `next(action)` which returns a new state after all reducers have processed the action. Docs: https://redux.js.org/advanced/middleware#the-final-approach

File: tests/unit/core/test_store.js (hunk 2 above)
Comment (kumar303) [reply tests/unit/core/test_store.js]: It simulates how Redux middleware works.

Comment (kumar303) [reply tests/unit/core/test_store.js]: Did the documentation help? It's a well defined interface so I don't think it will change.

**Comment by kumar303** [comment]
@bobsilverberg thanks for the review. I added some explanations but let me know if you think it could use a follow-up to make things more clear.

**Pull request merged**